### DISCUSSION TITLE
posts/2022/11/nobori-daiyu-lec/

### DISCUSSION THREAD

**Discussion by giscus** [naoki-urabe/my-blog#56] [General]
# posts/2022/11/nobori-daiyu-lec/

  

最近、登大遊さんの講演動画を視聴したのですが、内容がかなり良かったので自分の心に残ったことを記します。

https://nao-eng.com/posts/2022/11/nobori-daiyu-lec/

**Comment by EG-yamagami** [comment]
動画を見づらい環境でも最近はログミーというテキスト化サービスもあっていいですよね。講演から引っかけたことを調べたり、モチベーションの喚起につながることもよくあります。私の経験から知識を実務的に役立つものから身に着けていったら、体系的に足りない部分を補完するフェーズが交互にやってくると考えてます。
